tendrl/commons/jobs/__init__.py

```diff
@@ -51,9 +51,7 @@ def _run(self):
 
                         # Job routing
                         if raw_job.get("payload", {}).get("tags", []):
-                            NS.node_context = NS.node_context.load()
-                            tags = json.loads(NS.node_context.tags)
-                            if set(tags).isdisjoint(
```

Comment:
NodeContext object is in commons, it can be accessed by any backend component at NS.node_context. 

Is the current code causing any traceback? This change is not required